Snippets/Sqs/Sqs_1/Usage.cs

```diff
@@ -7,6 +7,7 @@ class Usage
         #region SqsTransport
 
         var transport = busConfiguration.UseTransport<SqsTransport>();
+        // bucket only required for messages larger than 256KB
```

Comment:
maybe clarify which parameter you referring to, e.g. "S3 bucket only required [...]"